### PR TITLE
add heading component back to the main component structure

### PR DESCRIPTION
**Pull request by cezaraugusto**

### CODE REVIEW

File: src/components/text/heading/style.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+// All feature-based text inherits from StyledSharedHeading
+// which excludes normal h1...h6 headings
+const StyledSharedHeading = styled.span`
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  -webkit-font-smoothing: antialiased;
+` as any
+
+const StyledH1 = StyledSharedHeading.withComponent('h1').extend`
```
Comment (NejcZdovc) [inline src/components/text/heading/style.ts]: Please set headers based on the design language.

h1 48px
h2 38px
h3 28px
h4 22px
h5 16px
h6 14px
h7 12px

File: src/components/text/heading/style.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (NejcZdovc) [inline src/components/text/heading/style.ts]: Font weight, letter-spacing and line-height variates between different headers. Please add them based on the design language

File: src/components/text/heading/style.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+`
+
+const StyledH5 = StyledSharedHeading.withComponent('h5').extend`
+  font-size: 16px;
+`
+
+const StyledH6 = StyledSharedHeading.withComponent('h6').extend`
+  font-size: 14px;
+`
+
```
Comment (NejcZdovc) [inline src/components/text/heading/style.ts]: H7 is missing

File: src/components/text/heading/style.ts (hunk 2 above)
Comment (cezaraugusto) [reply src/components/text/heading/style.ts]: ok talked with @rossmoody and we decided to leave letter-spacing/line-height open atm. new spec available at https://app.goabstract.com/projects/505d2110-74ea-11e8-b35b-eb800813dd0e/branches/master/commits/1e57e5e86759ba5add339fc07d17baa4beae372e/files/C9D94365-DAF9-4758-B1F6-9310D9C8157A/layers/D4259C65-D83D-4B35-BEEA-B7168CC3CA11?mode=design

Comment (cezaraugusto) [reply src/components/text/heading/style.ts]: this pr is updated with the above

Comment (NejcZdovc) [reply src/components/text/heading/style.ts]: thank you

**Pull request merged**